helpers/apps

```diff
@@ -111,3 +111,68 @@ ynh_remove_apps() {
 		done
 	fi
 }
+
+# Load an app environment in the current Bash shell
+#
+# usage: ynh_install_apps --app="app"
+#     | arg: -a, --app=     - the app ID
+#
+# Requires YunoHost version 11.0.* or higher.
```

Comment:
If I am correct, we could also suggest using this command as follow:

```suggestion
# examples:
#   ynh_load_app_environment --app="APP" <<< 'echo "$USER"'
#   ynh_load_app_environment --app="APP" < /tmp/some_script.bash
# 
# Requires YunoHost version 11.0.* or higher.
```

Reply:
Absolutely! :)